cypress-io/cypress
# test(system-tests): add a regression test for #23552

- Closes https://github.com/cypress-io/cypress/issues/23552

### User facing changelog


N/A

### Additional details


Fixed passively by https://github.com/cypress-io/cypress/pull/23637. We don't have any tests around `module: NodeNext` in `tsconfig.json` so I added one.

### Steps to test


N/A, just ensure CI is green. That is the test!

You could check out `git checkout b5ba6d7b87deaa8b5f275b20e38127b740490bfd` which is the commit before #23637 which fixed this, run `yarn install` and run the system test and watch it regress, if you really want.

### How has the user experience changed?


Works with `module: "NodeNext"`.

### PR Tasks
